### ISSUE TITLE
"Cross-posted" line confused me

### ISSUE THREAD

**Issue by hackergrrl** [yoshuawuyts/markdown-to-medium#2]
Hiya! I didn't notice this until today that the Medium article I posted using this module appended the line `*Cross-posted from [https://github.com/yoshuawuyts/writing](${canonicalUrl})*` ([src](https://github.com/yoshuawuyts/markdown-to-medium/blob/master/index.js#L47)).

I hadn't expected this behaviour, and found it a bit confusing. Would people think that my ariticle is a cross-post of something @yoshuawuyts wrote?

I personally feel a bit conflicted about the idea of a CLI tool inserting advertising for itself into its output, but I 100% respect this is your module, your preferences, etc.. If not removed, my only request would be to maybe rephrase the note to make its origin a bit clearer? What do you think?

**Comment by yoshuawuyts** [comment]
oh lol, no it was supposed to link to a blog or something you host yourself; treat medium as just another publication channel - perhaps we should turn that into an option or something and turn off by default - PR welcome!

**Comment by hackergrrl** [comment]
Ooooh! Ah ha, I see now! I completely misunderstood. :grinning: 

Perhaps a CLI option for a name + url? Something like

``` sh
$ markdown-to-medium foo.md -t "eight45.net blog" -u "http://blog.eight45.net/foo.html"
```

and just omit altogether if the options aren't present?

**Comment by yoshuawuyts** [comment]
yup sounds good to me

**Comment by RichardLitt** [comment]
This was closed in https://github.com/yoshuawuyts/markdown-to-medium/pull/3. 🎉
